manifest/kubernetes/rhel/ncp-rhel.yaml

```diff
@@ -1083,14 +1082,6 @@ spec:
         - name: dir-tmp-usr-ovs-kmod-backup
           hostPath:
             path: /tmp/nsx_usr_ovs_kmod_backup
-
-
-        - name: host-usr-share
-          hostPath:
-            path: /usr/share
-        - name: host-depmod
-          hostPath:
-            path: /sbin/depmod
 {{else}}
```

Comment:
we can just remove code line #L 1085 - 1095
{{else}}
...
#path: /tmp/nsx_user_ovs_kmod_backup